homeassistant/components/zha/number.py

```diff
@@ -947,3 +949,25 @@ class AqaraThermostatAwayTemp(ZHANumberConfigurationEntity):
     _attr_mode: NumberMode = NumberMode.SLIDER
     _attr_native_unit_of_measurement: str = UnitOfTemperature.CELSIUS
     _attr_icon: str = ICONS[0]
+
+
+@MULTI_MATCH(
```

Comment:
This should use CONFIG_DIAGNOSTIC_MATCH defined on line 38

Reply:
addressed with
https://github.com/home-assistant/core/pull/105765/commits/3435b63b82ce963cfc9c606cb9aadf91fe04a695